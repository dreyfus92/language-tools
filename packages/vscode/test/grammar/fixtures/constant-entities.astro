<p>Hello this is testing &nbsp;</p>

<p>Testing &quot; quoted part &quot;</p>
