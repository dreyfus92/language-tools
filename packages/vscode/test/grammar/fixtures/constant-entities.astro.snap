><p>Hello this is testing &nbsp;</p>
#^ source.astro meta.scope.tag.p.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^ source.astro meta.scope.tag.p.astro meta.tag.start.astro entity.name.tag.astro
#  ^ source.astro meta.scope.tag.p.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#   ^^^^^^^^^^^^^^^^^^^^^^ source.astro text.astro
#                         ^^^^^^ source.astro text.astro constant.entity.astro
#                               ^^ source.astro meta.scope.tag.p.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#                                 ^ source.astro meta.scope.tag.p.astro meta.tag.end.astro entity.name.tag.astro
#                                  ^ source.astro meta.scope.tag.p.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>
><p>Testing &quot; quoted part &quot;</p>
#^ source.astro meta.scope.tag.p.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^ source.astro meta.scope.tag.p.astro meta.tag.start.astro entity.name.tag.astro
#  ^ source.astro meta.scope.tag.p.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#   ^^^^^^^^ source.astro text.astro
#           ^^^^^^ source.astro text.astro constant.entity.astro
#                 ^^^^^^^^^^^^^ source.astro text.astro
#                              ^^^^^^ source.astro text.astro constant.entity.astro
#                                    ^^ source.astro meta.scope.tag.p.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#                                      ^ source.astro meta.scope.tag.p.astro meta.tag.end.astro entity.name.tag.astro
#                                       ^ source.astro meta.scope.tag.p.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>